ios/RNSentryBreadcrumbConverter.h

```diff
@@ -0,0 +1,13 @@
+@import Sentry;
+
+@class SentryRRWebEvent;
+
+@interface RNSentryBreadcrumbConverter
+    : NSObject <SentryReplayBreadcrumbConverter>
```

Comment:
The build is complaining that the `SentryReplayBreadcrumbConverter` should be a pointer.

Reply:
are you using the custom branch? https://github.com/getsentry/sentry-cocoa/pull/4027

Reply:
Ahh, my bad, ofc the CI is missing the branch.